rpcd/playbooks/group_vars/all.yml

```diff
@@ -28,3 +28,7 @@ migrations_dir: /etc/openstack_deploy/migrations
 
 # Ceph Conf Directory Mode
 conf_directory_mode: 755
+
+# This should default to the same default for the
+# cinder_service_backup_program_enabled variable in the os_cinder role.
+maas_monitor_cinder_backup: "{{ cinder_service_backup_program_enabled | default(false) }}"
```

Comment:
My preference is to explicitly set cinder_service_backup_program_enabled, the way this has been managed previously is by setting the OSA var and the rpc_maas var in the overrides files.